backtrace-library/src/main/java/backtraceio/library/services/BacktraceNativeCrashResponseListener.java

```diff
@@ -0,0 +1,127 @@
+package backtraceio.library.services;
+
+import android.util.Log;
+
+import java.io.File;
+import java.io.FilenameFilter;
+import java.util.ArrayList;
+import java.util.List;
+
+import backtraceio.library.BacktraceClient;
+import backtraceio.library.base.BacktraceBase;
+import backtraceio.library.events.OnServerResponseEventListener;
+import backtraceio.library.logger.BacktraceLogger;
+import backtraceio.library.models.BacktraceResult;
+import backtraceio.library.models.json.BacktraceReport;
+import backtraceio.library.models.types.BacktraceResultStatus;
+
+public class BacktraceNativeCrashResponseListener
+        implements OnServerResponseEventListener {
+    private static transient final String LOG_TAG = BacktraceNativeCrashResponseListener.
+            class.getSimpleName();
+
+    private static BacktraceNativeCrashResponseListener instance;
+    private static BacktraceClient client;
+    private static String databasePath;
```

Comment:
where you are setting this field?

Reply:
Removed the fields that aren't being set.